streams/views.py

```diff
@@ -32,3 +33,14 @@ def put(self,request):
         return Response(status=status.HTTP_401_UNAUTHORIZED) 
 
 
+class PostsByStreamView(APIView):
+
+    def get(self,request,pk):
+        try:
+            stream = Stream.objects.get(pk=pk)
+            serializer = PostByStreamSerializer(stream)
+            return Response(serializer.data)
+        except:
+            return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
Just wanted to confirm , did you check that on accessing a stream which does not exist the response is 404 not found ? @nike0503

Reply:
Yes I checked that